### PR TITLE
add init command

### PR DESCRIPTION
**Pull request by bketelsen**
The init command prepares the entire environment.  This is just a starting point, much more work needs to be done in the future to customize these to environmental differences (ethernet port name, etc)

### CODE REVIEW

**Comment by bketelsen** [comment]
@moonmeister @brauner Please take a look?

File: cmd/init.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+		}
+		log.Success("Default configuration file created")
+
+		// create templates directory
+
+		log.Running("Creating templates")
+		err = createTemplates()
+		if err != nil {
+			log.Error("Error creating templates: " + err.Error())
+		}
```
Comment (brauner) [inline cmd/init.go]: Should be an `os.Exit(1)` here, I guess or does `log.Error()` exit?

File: cmd/init.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+		log.Success("Templates created")
+		// create lxc profiles
+
+		log.Running("Creating lxc profiles")
+		err = createProfiles("")
+		if err != nil {
+			log.Error("Error creating profiles: " + err.Error())
```
Comment (brauner) [inline cmd/init.go]: Should be an `os.Exit(1)` here, I guess or does `log.Error()` exit?

**Comment by moonmeister** [comment]
@bketelsen Looks like a good place to start. I don't have enough go experience to comment on particular parts of the code.

**Pull request merged**